backend/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+CONTAINER_ALREADY_STARTED="CONTAINER_ALREADY_STARTED_PLACEHOLDER"
```

Comment:
Personally, I would prefer this file to be hidden (prefixed with `.`), but we will probably not see it anyway because we cannot really touch the container.

Reply:
This part is to initialize the local veriable

Reply:
This variable is then used as the file name, so it is also the file's name, right?